source/AI.cpp

```diff
@@ -2463,7 +2481,7 @@ void AI::DoMining(Ship &ship, Command &command)
 
 
 
-bool AI::DoHarvesting(Ship &ship, Command &command)
+bool AI::DoHarvesting(Ship &ship, Command &command) const
```

Comment:
Just wondering, why was this made const? That change should probably be removed and then I'm good with this PR